container-core/src/main/resources/configdefinitions/container.qr-searchers.def

```diff
@@ -64,7 +64,7 @@ searchcluster[].name string
 searchcluster[].searchdef[] string
 
 ## configid that may be used to get rank-profiles config for the cluster.
-searchcluster[].rankprofiles.configid reference default=""
+searchcluster[].rankprofiles_configid string default=""
```

Comment:
Any side effects of this change, or is `reference` and `string` more or like the same?

Reply:
Should be the same. Reference was pre-historic. Both @hmusum and @gjoranv recommended replacement.